ui/src/Groups/Messages/Messages.spec.js

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+import { ProducerMessages, ConsumerMessages } from './index.js';
+import {
+  CONSUMER,
+  PRODUCER,
+  STATUS_SUCCESS,
+  STATUS_ERROR,
+} from './Messages.assets.js';
+import { fireEvent, render } from 'TestUtils';
+
+describe('Messages Element component', () => {
+  const testClassName = 'testCssClass';
+  const testMessage = {
```

Comment:
This is me being picky, but could these message shapes be shared between the tests and storybook? Less code to have to update/we test the expected shape at test time and in storybook that way

Reply:
Done